kubernetes-maven-plugin/doc/src/main/asciidoc/inc/goals/develop/_jkube-log.adoc

```diff
@@ -42,6 +42,15 @@ mvn {goal-prefix}:log -Djkube.log.container=foo
 [[Supported-Properties-Log]]
 === Supported Properties for Log goal
 
+[source,xml,indent=0,subs="verbatim,quotes,attributes"]
+<plugin>
+  <configuration>
+    <logFollow>true</logFollow>
+    <logContainer>container</logContainer>
+    <logPod>pod</logPod>
+  </configuration>
+</plugin>
```

Comment:
This XML configuration example looks a bit out of place. It's between Supported Properties title and table body.

Reply:
Added a "Example logs configuration" that is separated from Supported Properties title.